### PR TITLE
Accessbility checkup

### PR DESCRIPTION
**Pull request by Salimer**
In this branch I improved the accessibility of my website by performing the following fixes: 

Page titles
- Updated the title to adequately and briefly describe the page's content. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L10-R10)

Image text alternatives
- Updated the logo image element to adequately and briefly describe the image's content. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L15-R15)
- Added the ITA to social media icons. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L37-R48)
- Added the image text alternative to the image element of the nav to adequately and briefly describes the content of the image. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L17-R17)
- Updated the bottom-arrow image element to adequately and briefly describe the image's content. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L51-R51)
- Added the ITA to the technologies’ card icons. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L204-R221)
 
Text headings
- Update text headings to match The heading hierarchy is meaningful. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L58-R173)

Color contrast - no issues found

Resize - no issues found

Interaction
- Added labels to form elements. [here](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R250-R270)

Moving content - no issues found

Multimedia - no issues found 

The basic structure of the page - no issues found

### CODE REVIEW

File: .history/index_20230403180958.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
```
Comment (AdnanOlarmmi) [inline .history/index_20230403180958.html:10]: - [ ] Kindly note that the title should satisfactorily and succinctly describe the page's content. As an example:
Bad titles:
My portfolio.
Portfolio.


Improved page titles:
John's Portfolio
John Front-end web developer

File: .history/index_20230403180958.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+</head>
+
+<body>
+    <header class="header">
+        <img id="logo" src="murple_logo.png" alt="My logo">
+        <nav>
+            <img class="Normal-Button" src="Assets/nav.svg">
```
Comment (AdnanOlarmmi) [inline .history/index_20230403180958.html:17]: - [ ] Kindly include an `alt` attribute here, indicating to a screen reader and a visually impaired person that this is a nav bar.

File: .history/index_20230403180958.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+                <span class="color4">Hey there, I'm Salim.<br></span>
+                <span class="color1">I'm a software developer</span>
+            </h1>
+            <p class="color4 Supporting-Text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hesitate to contact me.
+            </p>
+            <ul class="social-media-icons">
+                <li><a href="https://twitter.com/SaLiM355173"><img src="Assets/twitter.svg"></a></li>
+                <li><a href="https://www.linkedin.com/in/salim-bamahfoodh-707b74200/"><img src="Assets/Linkedin icon.svg"></a></li>
```
Comment (AdnanOlarmmi) [inline .history/index_20230403180958.html:38]: - [ ] Kindly include an `alt` attribute here, indicating to a screen reader and a visually impaired person what the image entails.

File: .history/index_20230403180958.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+                <li><a href="#"><img src="Assets/medium.svg"></a></li>
+                <li><a href="https://github.com/Salimer"><img src="Assets/github.svg"></a></li>
+                <li><a href="#"><img src="Assets/angellist.svg"></a></li>
+            </ul>
+            <ul class="social-media-icons-desktop">
+                <li><a href="https://github.com/Salimer"><img src="Assets/github.svg"></a></li>
+                <li><a href="https://www.linkedin.com/in/salim-bamahfoodh-707b74200/"><img src="Assets/Linkedin icon.svg"></a></li>
```
Comment (AdnanOlarmmi) [inline .history/index_20230403180958.html:45]: - [ ] Kindly include an `alt` attribute here, indicating to a screen reader and a visually impaired person what the image entails.

File: .history/index_20230403180958.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+                    </ul>
+                </form>
+                <p class="email color4">s.bamahfoodh@gmail.com</p>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <ul class="social-media-icons">
+            <li><a href="https://twitter.com/SaLiM355173"><img src="Assets/twitter.svg"></a></li>
+            <li><a href="https://www.linkedin.com/in/salim-bamahfoodh-707b74200/"><img src="Assets/Linkedin icon.svg"></a></li>
```
Comment (AdnanOlarmmi) [inline .history/index_20230403180958.html:275]: - [ ] Kindly include an `alt` attribute here, indicating to a screen reader and a visually impaired person what the image entails.

File: .history/index_20230403181545.html [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+            </ul>
+            <ul class="social-media-icons-desktop">
+                <li><a href="https://github.com/Salimer"><img src="Assets/github.svg"></a></li>
+                <li><a href="https://www.linkedin.com/in/salim-bamahfoodh-707b74200/"><img src="Assets/Linkedin icon.svg"></a></li>
+                <li><a href="#"><img src="Assets/angellist.svg"></a></li>
+                <li><a href="https://twitter.com/SaLiM355173"><img src="Assets/twitter.svg"></a></li>
+                <li><a href="#"><img src="Assets/medium.svg"></a></li>
+            </ul>
+            <div class="bottom-arrow">
+                <img src="Assets/bottom arrow.svg" alt="disabled" width="40px" height="40px">
```
Comment (AdnanOlarmmi) [inline .history/index_20230403181545.html:51]: - [ ] For these kinds, you should specify what type of image they are not to cause ambiguity for screen readers and visually impaired persons 😃. for example "Project Image" or more explanatory like "Calculator Project Image" or "Google clone project snippet".

This is applicable to all ambiguous image `alt` attributes not just this 😉.

- If not adding an `alt` won't affect screen readers kindly make the value for `alt` as `null`.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @Salimer, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice UI design.
- Good structure.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Kindly follow semantic HTML guidelines in your code 🙃.
You should have something like this instead of repetitive `div`s this enhances SEOs.
```
<body>
  <header>
    <nav>
    <nav>
  </header>
  
  <main>
    <section>
    </section>
    <article>
    </article>
  </main>
  
  <footer>
    <section>
    </section>
  </footer>
</body>
```

check this [link](https://web.dev/learn/html/semantic-html/) to read, know and learn more about semantic HTML. 💯 😀.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by waelelsafty07** [comment]
Hello @AdnanOlarmmi I'm a coding partner of @Salimer  
  You reviewed wrong file on [.history/index_20230403180958.html](https://github.com/Salimer/Portfolio/pull/9/files/ffa8c467f867a5762a85a198ce66bfa6a35e964a#diff-508d513507639581e3b70dec9414b625eb70e6921ec58a7e03d3ee4c4a9a4f0a) and this is old version 
you should have reviewed [index.html](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051).
All required changes are implemented in [index.html](https://github.com/Salimer/Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051) before @Salimer requested a review yesterday

**Comment by AdnanOlarmmi** [comment]
Hi @waelelsafty07 and @Salimer, My bad for making such mistake, kindly fix the changes applicable to the current file and ask for a re-review 😊

File: index.html [hunk 7]
```diff
@@ -34,28 +34,28 @@ <h1 class="Primary">
...
-                <li><a href="#"><img src="Assets/medium.svg"></a></li>
+                <li><a href="https://github.com/Salimer"><img src="Assets/github.svg" alt="GitHub"></a></li>
+                <li><a href="https://www.linkedin.com/in/salim-bamahfoodh-707b74200/"><img src="Assets/Linkedin icon.svg" alt="LinkedIn"></a></li>
+                <li><a href="#"><img src="Assets/angellist.svg" alt="Angellist"></a></li>
+                <li><a href="https://twitter.com/SaLiM355173"><img src="Assets/twitter.svg" alt="Twitter"></a></li>
+                <li><a href="#"><img src="Assets/medium.svg" alt="medium"></a></li>
             </ul>
             <div class="bottom-arrow">
-                <img src="Assets/bottom arrow.svg" alt="disabled" width="40px" height="40px">
+                <img src="Assets/bottom arrow.svg" alt="bottom-arrow" width="40px" height="40px">
```
Comment (OyePriscilla) [inline index.html:51]: - [optionally] Kindly keep all inline styling in a separate CSS file according to the best practices. 👍

**Review by OyePriscilla — APPROVED** [review]
Hi @Salimer 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Descriptive page title ✔️ 
Add alt attributes for images ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**